core/commands/destroy_command.rb

```diff
@@ -50,6 +51,12 @@ def show_help
 Or you can destroy all nodes:
   mdbci destroy configuration
 
+You can destroy nodes by name without the need for configuration file:
+  mdbci destroy --node-name
```

Comment:
This example is not complete. The `--node-name` requires a parameter to be present.

The description also lacks information about what part of node name should be specified by the end user.